docs/man.md

```diff
@@ -93,7 +90,7 @@ If you choose the offline option to run the install command, you must do the fol
 1. Run qpc-tools with the required options to complete an offline installation.  For example:
 
     ```
-    qpc-tools server install --offline --offline-files='/PATH' --version=0.9.1
+    qpc-tools server install --password --dbms-password --offline --offline-files='/PATH' --version=0.9.1
```

Comment:
Why do we call it `dbms-password`?

Reply:
It stands for database management system _i think_ (i'm not sure the original intention in using dbms) - but I changed it to `db-password` to make it more clear.